src/odemis/gui/cont/streams.py

```diff
@@ -3383,6 +3403,14 @@ def _on_button(self, evt):
             roc_ctrl = self.roc_ctrls[num]
         roc_ctrl.fit_view_to_bbox()
 
+    @call_in_wx_main
+    def _on_active_scintillators(self, evt):
```

Comment:
This is called when the active_scintillators VA changes, right? Then the argument is not "evt". It should be active_sincillators (or shorter), and you can use it, instead of reading the value again in the loop.